plasmapy/formulary/dimensionless.py

```diff
@@ -157,3 +160,69 @@ def Reynolds_number(rho: u.kg / u.m ** 3, v: u.m / u.s, L: u.m,
     """
     Re = abs(rho * v * L / mu)
     return Re
+
+
+@validate_quantities(U={"can_be_negative": True})
+def Mag_Reynolds(U: u.m / u.s, L: u.m, sigma: u.S / u.m) -> u.dimensionless_unscaled:
+    r"""
+    The Magnetic Reynolds number is a dimensionless quantity that
+    estimates the relative contributions of advection and induction
+    to magnetic diffusion in a conducting medium.
+
+    .. math::
+
+        Rm = \frac{U L}{\\eta}
+
+        where: \eta = \frac{1}{\mu_0 \sigma}
+
+        and \mu_0 is the permeability of free space.
+
+    Parameters
+    ----------
+    U : `~astropy.units.Quantity`
+        The velocity scale of the plasma.
```

Comment:
This "velocity scale" seems to be the same concept as as `v` - the "flow velocity" - for `Reynolds_number`, no? It'd probably make sense to keep the same variable name.

Reply:
ok, I changed the v to a U in `Reynolds_number` along with fixing some small formatting mistakes